tokens/ocean.svg

```diff
@@ -0,0 +1 @@
875 -2.625 C 15.332031 -0.875 12.867188 0 9.828125 0 L 1.890625 0 Z M 6.703125 -15.0625 L 6.703125 -3.828125 L 10.125 -3.828125 C 11.601562 -3.828125 12.8125 -4.34375 13.75 -5.375 C 14.695312 -6.414062 15.171875 -7.765625 15.171875 -9.421875 C 15.171875 -11.078125 14.675781 -12.429688 13.6875 -13.484375 C 12.695312 -14.535156 11.4375 -15.0625 9.90625 -15.0625 Z M 6.703125 -15.0625" transform="translate(132.914 275.854)"/></g></g></g><g fill="#FFF"><g><g><path d="M 16.328125 -15.03125 L 10.640625 -15.03125 L 10.640625 0 L 5.828125 0 L 5.828125 -15.03125 L 0.1875 -15.03125 L 0.1875 -18.921875 L 16.328125 -18.921875 Z M 16.328125 -15.03125" transform="translate(177.113 275.854)"/></g></g></g><g fill="#FFF"><g><g><path d="M 1.640625 -3.671875 C 2.054688 -4.085938 2.597656 -4.296875 3.265625 -4.296875 C 3.929688 -4.296875 4.472656 -4.085938 4.890625 -3.671875 C 5.316406 -3.253906 5.53125 -2.710938 5.53125 -2.046875 C 5.53125 -1.378906 5.316406 -0.832031 4.890625 -0.40625 C 4.472656 0.0078125 3.929688 0.21875 3.265625 0.21875 C 2.597656 0.21875 2.054688 0.0078125 1.640625 -0.40625 C 1.234375 -0.832031 1.03125 -1.378906 1.03125 -2.046875 C 1.03125 -2.710938 1.234375 -3.253906 1.640625 -3.671875 Z M 1.640625 -3.671875" transform="translate(217.1 275.854)"/></g></g></g><g fill="#FFF"><g><g><path d="M 22.65625 0 L 18.40625 0 L 18.390625 -12.28125 L 13.84375 -1.890625 L 10.71875 -1.890625 L 6.1875 -12.28125 L 6.1875 0 L 1.890625 0 L 1.890625 -18.921875 L 7.15625 -18.921875 L 12.28125 -7.671875 L 17.421875 -18.921875 L 22.65625 -18.921875 Z M 22.65625 0" transform="translate(225.011 275.854)"/></g></g></g><g fill="#FFF"><g><g><path d="M 13.28125 0 L 10.46875 -5.3125 L 6.703125 -5.3125 L 6.703125 0 L 1.890625 0 L 1.890625 -18.90625 L 10.34375 -18.90625 C 12.84375 -18.90625 14.773438 -18.332031 16.140625 -17.1875 C 17.515625 -16.039062 18.203125 -14.414062 18.203125 -12.3125 C 18.203125 -9.394531 17.054688 -7.367188 14.765625 -6.234375 L 18.734375 0 Z M 6.703125 -9.09375 L 10.34375 -9.09375 C 11.382812 -9.09375 12.191406 -9.351562 12.765625 -9.875 C 13.347656 -10.40625 13.640625 -11.164062 13.640625 -12.15625 C 13.640625 -13.125 13.347656 -13.859375 12.765625 -14.359375 C 12.191406 -14.867188 11.382812 -15.125 10.34375 -15.125 L 6.703125 -15.125 Z M 6.703125 -9.09375" transform="translate(273.071 275.854)"/></g></g></g><g fill="#FFF"><g><g><path d="M 14.203125 0 L 9.203125 -7.421875 L 6.703125 -4.640625 L 6.703125 0 L 1.890625 0 L 1.890625 -18.90625 L 6.703125 -18.90625 L 6.703125 -10.71875 L 13.9375 -18.90625 L 19.5 -18.90625 L 12.46875 -11.0625 L 19.953125 0 Z M 14.203125 0" transform="translate(293.97 275.854)"/></g></g></g><g fill="#FFF"><g><g><path d="M 16.6875 -15.125 L 6.703125 -15.125 L 6.703125 -11.359375 L 15.71875 -11.359375 L 15.71875 -7.59375 L 6.703125 -7.59375 L 6.703125 -3.78125 L 16.984375 -3.78125 L 16.984375 0 L 1.890625 0 L 1.890625 -18.90625 L 16.6875 -18.90625 Z M 16.6875 -15.125" transform="translate(315.488 275.854)"/></g></g></g></svg>
```

Comment:
The icon should be less than 10 kB (recommended is 5-7 kB), moreover, it should match an official token image (if there's any). And this one is completely different and seems to be referring to some other project.

```suggestion
<svg xmlns="[http://www.w3.org/2000/svg]()" width="[512]()" height="[512]()" viewBox="[0 0 512 512]()">
  <g fill="[#141414]()" fill-rule="[evenodd]()" transform="[translate(72)]()">
    <path d="[M61.6189076 477.44077L61.6189076 477.176261C61.6189076 461.142241 49.5855856 448.368888 34.150703 448.368888 18.3201779 448.368888 6.93339364 461.276899 6.93339364 476.926181L6.93339364 477.176261C6.93339364 493.219901 18.9715406 505.993254 34.4015983 505.993254 50.2369482 505.993254 61.6189076 493.090051 61.6189076 477.44077M0 477.44077L0 477.176261C0 458.540439 14.5229749 442.376569 34.4015983 442.376569 54.1692487 442.376569 68.5523013 458.28074 68.5523013 476.926181L68.5523013 477.176261C68.5523013 495.821702 54.0293264 512 34.150703 512 14.3927023 512 0 496.08621 0 477.44077M83.5481172 477.44077L83.5481172 477.176261C83.5481172 458.540439 98.4219452 442.376569 117.467759 442.376569 129.988529 442.376569 137.81762 448.104379 144.082817 454.630524L139.516248 459.309919C133.775557 453.457068 127.245701 448.368888 117.333024 448.368888 102.204161 448.368888 90.4629314 461.012391 90.4629314 476.926181L90.4629314 477.176261C90.4629314 493.219901 102.593931 505.993254 117.85753 505.993254 127.245701 505.993254 134.550286 501.1792 140.165866 494.912755L144.60251 498.82748C137.81762 506.512652 129.72387 512 117.467759 512 98.2872097 512 83.5481172 496.08621 83.5481172 477.44077M211.593719 474.050253C210.679188 460.627651 202.982689 448.234229 187.714838 448.234229 174.526341 448.234229 164.355795 459.314728 163.176532 474.050253L211.593719 474.050253zM156.384937 477.320539L156.384937 477.046412C156.384937 457.751723 169.968127 442.376569 187.969944 442.376569 206.635999 442.376569 218.51046 457.496833 218.51046 477.176261 218.51046 478.354526 218.51046 478.748884 218.380501 479.787681L163.176532 479.787681C164.355795 496.350719 176.22063 505.998063 189.28398 505.998063 199.579672 505.998063 206.635999 501.438899 212.108744 495.571621L216.546626 499.486346C209.759844 506.786779 201.808239 512 189.019248 512 171.532456 512 156.384937 498.043573 156.384937 477.320539zM283.017988 485.33581L283.017988 477.704319C277.369168 476.265237 269.882663 474.695331 260.286943 474.695331 246.084889 474.695331 238.195937 480.99434 238.195937 490.321718L238.195937 490.583369C238.195937 500.303225 247.394058 505.952951 257.658909 505.952951 271.327598 505.952951 283.017988 497.551042 283.017988 485.33581M231.364017 490.971001L231.364017 490.714195C231.364017 476.783694 243.321089 468.905088 260.684543 468.905088 270.008732 468.905088 276.45275 470.092209 282.896768 471.807477L282.896768 468.905088C282.896768 455.367064 274.605368 448.413927 260.810611 448.413927 252.650127 448.413927 245.953972 450.652498 239.776637 453.806848L237.405587 448.14743C244.770872 444.74112 252.00524 442.376569 261.20821 442.376569 270.270565 442.376569 277.500085 444.8671 282.368252 449.731873 286.83882 454.199324 289.205021 460.372353 289.205021 468.648282L289.205021 510.420402 282.896768 510.420402 282.896768 499.251775C278.295283 505.686455 270.008732 512 257.261309 512 244.770872 512 231.364017 504.906347 231.364017 490.971001M311.698745 443.946247L318.075714 443.946247 318.075714 456.421313C322.478745 448.534164 329.885802 442.376569 341.968984 442.376569 358.718708 442.376569 368.468619 453.931144 368.468619 469.957364L368.468619 510.92887 362.106024 510.92887 362.106024 471.139467C362.106024 457.346648 354.440246 448.277396 341.063464 448.277396 328.069971 448.277396 318.075714 458.267138 318.075714 472.181075L318.075714 510.92887 311.698745 510.92887 311.698745 443.946247zM47.1297071 179.409443C47.1297071 190.352909 38.2646994 199.230126 27.3138075 199.230126 16.3724839 199.230126 7.49790795 190.352909 7.49790795 179.409443 7.49790795 168.465977 16.3724839 159.598326 27.3138075 159.598326 38.2646994 159.598326 47.1297071 168.465977 47.1297071 179.409443M362.041841 179.409443C362.041841 190.352909 353.167265 199.230126 342.230726 199.230126 331.284618 199.230126 322.410042 190.352909 322.410042 179.409443 322.410042 168.465977 331.284618 159.598326 342.230726 159.598326 353.167265 159.598326 362.041841 168.465977 362.041841 179.409443M204.585774 257.604303C204.585774 268.546448 195.712269 277.422594 184.772266 277.422594 173.82748 277.422594 164.953975 268.546448 164.953975 257.604303 164.953975 246.662158 173.82748 237.790795 184.772266 237.790795 195.712269 237.790795 204.585774 246.662158 204.585774 257.604303M125.322176 219.046025C125.322176 229.984708 116.453454 238.861925 105.503884 238.861925 94.5590978 238.861925 85.6903766 229.984708 85.6903766 219.046025 85.6903766 208.097776 94.5590978 199.230126 105.503884 199.230126 116.453454 199.230126 125.322176 208.097776 125.322176 219.046025M282.778243 219.046025C282.778243 229.984708 273.904738 238.861925 262.964735 238.861925 252.015165 238.861925 243.146444 229.984708 243.146444 219.046025 243.146444 208.097776 252.015165 199.230126 262.964735 199.230126 273.904738 199.230126 282.778243 208.097776 282.778243 219.046025M40.7029289 239.402311C40.7029289 246.783926 34.7136299 252.786611 27.321041 252.786611 19.9188075 252.786611 13.9246862 246.783926 13.9246862 239.402311 13.9246862 232.00141 19.9188075 226.008368 27.321041 226.008368 34.7136299 226.008368 40.7029289 232.00141 40.7029289 239.402311M355.615063 239.402311C355.615063 246.783926 349.617199 252.786611 342.225941 252.786611 334.829862 252.786611 328.83682 246.783926 328.83682 239.402311 328.83682 232.00141 334.829862 226.008368 342.225941 226.008368 349.617199 226.008368 355.615063 232.00141 355.615063 239.402311M198.158996 318.651445C198.158996 326.047524 192.158472 332.050209 184.774694 332.050209 177.386097 332.050209 171.380753 326.047524 171.380753 318.651445 171.380753 311.260187 177.386097 305.271967 184.774694 305.271967 192.158472 305.271967 198.158996 311.260187 198.158996 318.651445M118.895397 279.034112C118.895397 286.428032 112.902355 292.41841 105.511098 292.41841 98.1150183 292.41841 92.1171548 286.428032 92.1171548 279.034112 92.1171548 271.635368 98.1150183 265.640167 105.511098 265.640167 112.902355 265.640167 118.895397 271.635368 118.895397 279.034112M276.351464 279.034112C276.351464 286.428032 270.34986 292.41841 262.969574 292.41841 255.570006 292.41841 249.573222 286.428032 249.573222 279.034112 249.573222 271.635368 255.570006 265.640167 262.969574 265.640167 270.34986 265.640167 276.351464 271.635368 276.351464 279.034112M34.2761506 286.519798C34.2761506 290.372146 31.1609642 293.48954 27.3162721 293.48954 23.4666509 293.48954 20.3514644 290.372146 20.3514644 286.519798 20.3514644 282.677314 23.4666509 279.564854 27.3162721 279.564854 31.1609642 279.564854 34.2761506 282.677314 34.2761506 286.519798M349.188285 286.519798C349.188285 290.372146 346.065958 293.48954 342.228408 293.48954 338.376059 293.48954 335.263598 290.372146 335.263598 286.519798 335.263598 282.677314 338.376059 279.564854 342.228408 279.564854 346.065958 279.564854 349.188285 282.677314 349.188285 286.519798M191.732218 365.785864C191.732218 369.631918 188.609892 372.753138 184.772341 372.753138 180.924925 372.753138 177.807531 369.631918 177.807531 365.785864 177.807531 361.939811 180.924925 358.828452 184.772341 358.828452 188.609892 358.828452 191.732218 361.939811 191.732218 365.785864M112.468619 326.158996C112.468619 330.000119 109.348504 333.121339 105.503812 333.121339 101.664049 333.121339 98.5439331 330.000119 98.5439331 326.158996 98.5439331 322.308011 101.664049 319.196653 105.503812 319.196653 109.348504 319.196653 112.468619 322.308011 112.468619 326.158996M269.924686 326.158996C269.924686 330.000119 266.80236 333.121339 262.964809 333.121339 259.107528 333.121339 256 330.000119 256 326.158996 256 322.308011 259.107528 319.196653 262.964809 319.196653 266.80236 319.196653 269.924686 322.308011 269.924686 326.158996M211.012552 26.7787227C211.012552 41.571099 199.021506 53.5564854 184.24151 53.5564854 169.447113 53.5564854 157.456067 41.571099 157.456067 26.7787227 157.456067 11.9901863 169.447113 0 184.24151 0 199.021506 0 211.012552 11.9901863 211.012552 26.7787227M211.012552 106.041841C211.012552 120.832261 199.021506 132.820084 184.24151 132.820084 169.447113 132.820084 157.456067 120.832261 157.456067 106.041841 157.456067 91.2562199 169.447113 79.2635983 184.24151 79.2635983 199.021506 79.2635983 211.012552 91.2562199 211.012552 106.041841M211.012552 184.239109C211.012552 199.022581 199.021506 211.012552 184.24151 211.012552 169.447113 211.012552 157.456067 199.022581 157.456067 184.239109 157.456067 169.446038 169.447113 157.456067 184.24151 157.456067 199.021506 157.456067 211.012552 169.446038 211.012552 184.239109M368.468619 106.041841C368.468619 120.832261 356.479722 132.820084 341.692776 132.820084 326.901031 132.820084 314.912134 120.832261 314.912134 106.041841 314.912134 91.2562199 326.901031 79.2635983 341.692776 79.2635983 356.479722 79.2635983 368.468619 91.2562199 368.468619 106.041841M53.5564854 106.041841C53.5564854 120.832261 41.5702396 132.820084 26.7806428 132.820084 11.991046 132.820084 0 120.832261 0 106.041841 0 91.2562199 11.991046 79.2635983 26.7806428 79.2635983 41.5702396 79.2635983 53.5564854 91.2562199 53.5564854 106.041841M132.820084 66.4100418C132.820084 81.1983132 120.832764 93.1882845 106.041841 93.1882845 91.2509185 93.1882845 79.2635983 81.1983132 79.2635983 66.4100418 79.2635983 51.6121709 91.2509185 39.6317992 106.041841 39.6317992 120.832764 39.6317992 132.820084 51.6121709 132.820084 66.4100418M132.820084 144.60011C132.820084 159.389707 120.832764 171.380753 106.041841 171.380753 91.2509185 171.380753 79.2635983 159.389707 79.2635983 144.60011 79.2635983 129.815314 91.2509185 117.824268 106.041841 117.824268 120.832764 117.824268 132.820084 129.815314 132.820084 144.60011M290.276151 66.4100418C290.276151 81.1983132 278.285105 93.1882845 263.495508 93.1882845 248.705911 93.1882845 236.719665 81.1983132 236.719665 66.4100418 236.719665 51.6121709 248.705911 39.6317992 263.495508 39.6317992 278.285105 39.6317992 290.276151 51.6121709 290.276151 66.4100418M290.276151 144.60011C290.276151 159.389707 278.285105 171.380753 263.495508 171.380753 248.705911 171.380753 236.719665 159.389707 236.719665 144.60011 236.719665 129.815314 248.705911 117.824268 263.495508 117.824268 278.285105 117.824268 290.276151 129.815314 290.276151 144.60011]()"/>
  </g>
</svg>
```